File: src/main/java/com/prmncr/normativecontrol/dbos/DocumentFile.java
package com.prmncr.normativecontrol.dbos;

import lombok.AllArgsConstructor;
import lombok.Getter;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Lob;
import javax.persistence.Table;

@Getter
@AllArgsConstructor
@Table
@Entity
public class DocumentFile {
    @Id
    private String id;
    @Lob
    private byte[] file;

    protected DocumentFile() {

    }
}

File: src/main/kotlin/com/maeasoftworks/normativecontrol/daos/DocumentFile.kt
package com.maeasoftworks.normativecontrol.daos


import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Lob
import javax.persistence.Table

@Table
@Entity
class DocumentFile(
    @Id
    val id: String,
    @Lob
    val file: ByteArray
)